### ISSUE TITLE
select dropdown display issue when using <html lang="zh">

### ISSUE THREAD

**Issue by lgyaxx** [twbs/bootstrap#22282]
bootstrap version: bootstrap-4.0.0-alpha.6-dist
browser: chrome 57.0.2987.110 (64-bit)

While using \<html lang="en"\>, everything works as expected, but when switched to \<html lang="zh"\>,
the form select dropdown aligns text below the normal position, making letters like "y", "g” partially blocked by the lower edge of the display block:

            <form action="{{ route('orders.store') }}" method="POST">
                <div class="form-group">
                    <label for="rooms">Rooms</label>
                    <select class="form-control" id="rooms">
                        @foreach ($rooms as $room)
                            <option>{{ $room->name }}</option>
                        @endforeach
                    </select>
                </div>
             </form>
The view looks like this:
![dropdown bug](https://cloud.githubusercontent.com/assets/25860525/24346295/3c578076-1306-11e7-9ee1-de8fbbe32318.jpg)

Adding a "custom-select" class to select would fix this:

\<select class="form-control custom-select" id="rooms"\>

![bug fix](https://cloud.githubusercontent.com/assets/25860525/24346380/a8846016-1306-11e7-91f3-818dcb6761ad.jpg)

CodePen Live Demo:
<p data-height="265" data-theme-id="0" data-slug-hash="qrMdME" data-default-tab="html,result" data-user="lgyaxx" data-embed-version="2" data-pen-title="qrMdME" class="codepen">See the Pen <a href="https://codepen.io/lgyaxx/pen/qrMdME/">qrMdME</a> by Eagle (<a href="http://codepen.io/lgyaxx">@lgyaxx</a>) on <a href="http://codepen.io">CodePen</a>.</p>

**Comment by Johann-S** [comment]
Bug reports must include a live demo of the problem. Per our contributing guidelines, please create a reduced test case via [JS Bin](https://jsbin.com/) or [CodePen](http://codepen.io/) and report back with your link.

**Comment by lgyaxx** [comment]
<p data-height="265" data-theme-id="0" data-slug-hash="qrMdME" data-default-tab="html,result" data-user="lgyaxx" data-embed-version="2" data-pen-title="qrMdME" class="codepen">See the Pen <a href="https://codepen.io/lgyaxx/pen/qrMdME/">qrMdME</a> by Eagle (<a href="http://codepen.io/lgyaxx">@lgyaxx</a>) on <a href="http://codepen.io">CodePen</a>.</p>
<script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>

**Comment by oktalk** [comment]
It's worth noting that this issue seems to only be reproducible on Windows. I've tested this on the latest Chrome, FireFox, and Safari with a Mac and PC.

**Comment by infinnie** [comment]
The default font-family for the lang `zh` is Microsoft YaHei, which, like Segoe UI, has a lower baseline than some other font families and therefore needs a larger line-height.

See JSFiddle: https://jsfiddle.net/8e6otjs3/

**Comment by mdo** [comment]
Inclined to close this as a won't fix given it very likely means rewriting all our input and button styles again to change the `line-height` and vertical `padding` values there. Changing just the `line-height` has no affect in Win10 Edge; we'd have to tweak the `padding`, too.

**Comment by mdo** [comment]
Closing per last comment.

**Comment by infinnie** [comment]
But that padding change would only be required for `select`s, since other elements would not be affected by that clipping-content behavior.
